Content.Server/GameTicking/GameTicker.cs

```diff
@@ -260,16 +264,20 @@ public void SetStartPreset(Type type)
             UpdateInfoText();
         }
 
-        private IEntity _spawnPlayerMob()
+        private IEntity _spawnPlayerMob(Job job)
         {
             var entity = _entityManager.SpawnEntityAt(PlayerPrototypeName, _getLateJoinSpawnPoint());
             if (entity.TryGetComponent(out InventoryComponent inventory))
             {
-                var uniform = _entityManager.SpawnEntity("UniformAssistant");
-                inventory.Equip(EquipmentSlotDefines.Slots.INNERCLOTHING, uniform.GetComponent<ClothingComponent>());
+                var gear = _prototypeManager.Index<StartingGearPrototype>(job.StartingGear).Equipment;
 
-                var shoes = _entityManager.SpawnEntity("ShoesBlack");
-                inventory.Equip(EquipmentSlotDefines.Slots.SHOES, shoes.GetComponent<ClothingComponent>());
+                foreach (var slotEquipementPair in gear)
+                {
+                    EquipmentSlotDefines.Slots slot;
+                    Enum.TryParse(slotEquipementPair.Key.ToUpper(), out slot);
```

Comment:
Need to add a if check here to make sure the slot is valid.

You can also put the variable definition inside the TryParse if you want.
`Enum.TryParse(slotEquipementPair.Key.ToUpper(), out EquipmentSlotDefines.Slots slot);`

Reply:
Is just logging the error sufficient? I mean there isn't much more that we can do I think.

Reply:
Test what happens when you put a invalid input. Usually you want it to continue and not crash the program, so might need a `continue` in addition to the error log.